app/models/split.rb

```diff
@@ -19,6 +19,10 @@ class Split < ActiveRecord::Base
 
   scope :active, -> { where(finished_at: nil) }
 
+  def has_details?
+    %w(hypothesis assignment_criteria description squad_owner).any? { |attr| send(attr).present? }
```

Comment:
minor nit: we can use `public_send` here because these are all public attributes.